galsim/chromatic.py

```diff
@@ -164,10 +384,24 @@ def drawImage(self, bandpass, image=None, integrator=None, **kwargs):
                                 for details.)  [default: None]
         @param integrator       One of the image integrators from galsim.integ [default: None,
                                 which will try to select an appropriate integrator automatically.]
+                                This keyword is ignored in the case that optimized interpolation
+                                between precomputed images is to be used to render the image.
```

Comment:
Or, if trapezoid is possible for the interpolated version, perhaps we can change this keyword to be a string, much like how we allow `x_interp` and `k_interp` to be strings rather than actual `Interpolation` objects.  

Then the user could more simply write `integrator='midpoint'` or `integrator='trapezoid'`, which would create the appropriate thing from galsim.integ in the normal case or use the appropriate weights in the interpolation case.  That seems a bit more user-friendly.